src/Array.java

```diff
@@ -0,0 +1,24 @@
+import java.io.File;
+// in this task when I change "divider" to 0, ArithmeticException is caught
+public class Array {
+    public static void arrayMethod() {
+        int[] array1;
+        array1 = new int[3];
+        array1[0] = 9;
```

Comment:
Specific values should not be in a method because the method becomes not universal

Reply:
void devider(int numbers[], int deviderValue) {
    int devidedResults[] = new int[numbers.length];
    try {
      for (int i = 0; i < numbers.length; i++) {
        int result = numbers[i] / deviderValue;
        devidedResults[i] = result;
      }
      System.out.println(Arrays.toString(devidedResults));
    } catch (ArithmeticException exception) {
      System.out.println("Back to school and learn math");
    }
  }